Sources/SWBProtocol/ProjectModel/Project.swift

```diff
@@ -54,13 +56,14 @@ extension Project: Serializable {
             serializer.serialize(buildConfigurations)
             serializer.serialize(defaultConfigurationName)
             serializer.serialize(developmentRegion)
+            serializer.serialize(knownLocalizations)
             serializer.serialize(classPrefix)
             serializer.serialize(appPreferencesBuildSettings)
         }
     }
 
     public init(from deserializer: any Deserializer) throws {
-        try deserializer.beginAggregate(11)
```

Comment:
For compatibility this likely needs to remain able to decode messages without the new field